n8n/custom-nodes/n8n-nodes-rabbitmq-enhanced/nodes/RabbitMqEnhanced/RabbitMqEnhanced.node.ts

```
@@ -814,7 +840,16 @@ export class RabbitMqEnhanced implements INodeType {
                   options.contentEncoding = 'gzip';
                   break;
                 case 'encrypt':
-                  messageContent = await transformer.encrypt(messageContent, 'n8n-secret-key');
+                  const encryptionKeyPublish = this.getNodeParameter('encryptionKey', i) as string;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)